frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -521,7 +531,7 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
             <K8sResourceSelectRow
               key="storage-class"
               id={asId('storage-class')}
-              isDisabled={isDisabled('storageClass')}
+              isDisabled={isDisabled('storageClass') || !isLoaded(storageClasses)}
```

Comment:
since this can affect volume/access modes it should wait also for storageClassConfigMap

we should also do the same thing for access/volume modes fields. 

It would be nice to have a common variable for that, or we could add if/switch statement to the isDisabled function and do the same thing by fieldName.

Reply:
done